src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -770,19 +769,20 @@ class DocBaseViewer extends BaseViewer {
      */
     initPdfViewerClass(PdfViewerClass) {
         const { file, location } = this.options;
+        const { AnnotationMode: PDFAnnotationMode = {} } = this.pdfjsLib;
         const assetUrlCreator = createAssetUrlCreator(location);
         const hasDownload = checkPermission(file, PERMISSION_DOWNLOAD);
         const hasTextLayer = hasDownload && !this.getViewerOption('disableTextLayer');
         const textLayerMode = this.isMobile ? PDFJS_TEXT_LAYER_MODE.ENABLE : PDFJS_TEXT_LAYER_MODE.ENABLE_ENHANCE;
 
         return new PdfViewerClass({
+            annotationMode: PDFAnnotationMode.ENABLE, // Show annotations, but not forms
             container: this.docEl,
             eventBus: this.pdfEventBus,
             findController: this.pdfFindController,
-            imageResourcesPath: assetUrlCreator(`third-party/doc/${DOC_STATIC_ASSETS_VERSION}/images/`),
+            imageResourcesPath: assetUrlCreator(IMAGES),
             linkService: this.pdfLinkService,
             maxCanvasPixels: this.isMobile ? MOBILE_MAX_CANVAS_SIZE : -1,
-            renderInteractiveForms: false, // Enabling prevents unverified signatures from being displayed
```

Comment:
Is this no longer relevant?

Reply:
It was replaced by the `annotationMode` option.

Reply:
@karelee7, I think we'll need to add this back in for IE11

Reply:
Should we add this back for legacy pdfjs?

Reply:
Added this line back in!